src/utils/allowlist-tokens.ts

```diff
@@ -1,4 +1,15 @@
-export const whitelistedTokens = {
+export type AllowlistedToken = {
+    spec?: string;
+    name: string;
+    symbol: string;
+    icon?: string;
+    reference?: string | null;
+    reference_hash?: string | null;
```

Comment:
Do we explicitly want string or null or undefined? It's our hard coded list, I get we could leave null out here. 
```suggestion
    reference?: string;
    reference_hash?: string;
```

Reply:
I went and looked at the list. Some of these values are set to null and could maybe be just removed. Some are just empty strings...

Reply:
Yes just matching with the types in the JSON, we don't use those fields particularly, just the whole object so leaving as is